src/v/config/configuration.h

```diff
@@ -228,6 +228,8 @@ struct configuration final : public config_store {
     // health monitor
     property<std::chrono::milliseconds> health_monitor_tick_interval;
     property<std::chrono::milliseconds> health_monitor_max_metadata_age;
+    property<unsigned> storage_space_alert_free_threshold_percent;
```

Comment:
This would be a good place to use the shiny new `bounded_property`, as percentages have a nice well defined min and max.

Reply:
IKR. There is a TODO to that effect in refresh_configuration. Planned as a followup patch.

Reply:
It's probably not worth punting to a separate PR -- using bounded_property should be pretty much a one-line change for each of these, to use the class in the header and then add some bounds in the constructor call.  I find it's easy to track this kind of thing in the context of an open PR than to remember the TODO later after the PR is merged.

Reply:
Let's get the bounds into this PR please -- it doesn't make sense to me to add the option here but add the validation bounds in a separate PR later.  Should only take a minute.

Reply:
Sorry, I thought I already said that I was going to add it as an additional patch above? There is one more coming for removing the existing ad-hoc bounds.